private-templates-service/adaptivecards-templating-service/src/models/models.ts

```diff
@@ -0,0 +1,27 @@
+export interface IUser {
```

Comment:
I think these are duplicates, Seva had some similar in his PR that went in today.  You might be able to remove some of these now

Reply:
Oh you have deleted his 👍